context/ue.go

```diff
@@ -20,13 +20,10 @@ import (
 // key is supi
 type UeContext struct {
 	// Ue Context
-	Supi                      string
-	Gpsi                      string
-	Pei                       string
-	GroupIds                  []string
-	PolAssociationIDGenerator uint32
-	AMPolicyData              map[string]*UeAMPolicyData // use PolAssoId(ue.Supi-numPolId) as key
-
+	Supi         string
+	Gpsi         string
+	Pei          string
+	AMPolicyData map[string]*UeAMPolicyData // use PolAssoId(ue.Supi-numPolId) as key
```

Comment:
What about doing something like this to keep all the variable related to the comment (`Ue Context`) together instead of splitting its variables to different locations?
```diff
diff --git a/context/ue.go b/context/ue.go
index 3acfa26..a07bdd1 100644
--- a/context/ue.go
+++ b/context/ue.go
@@ -19,14 +19,6 @@ import (

 // key is supi
 type UeContext struct {
-       // Ue Context
-       Supi                      string
-       Gpsi                      string
-       Pei                       string
-       GroupIds                  []string
-       PolAssociationIDGenerator uint32
-       AMPolicyData              map[string]*UeAMPolicyData // use PolAssoId(ue.Supi-numPolId) as key
-
        // Udr Ref
        UdrUri string
        // SMPolicy
@@ -41,6 +33,14 @@ type UeContext struct {
        AppSessionIdStore           *AppSessionIdStore
        PolicyDataSubscriptionStore *models.PolicyDataSubscription
        PolicyDataChangeStore       *models.PolicyDataChangeNotification
+
+       // Ue Context
+       Supi                      string
+       Gpsi                      string
+       Pei                       string
+       AMPolicyData              map[string]*UeAMPolicyData // use PolAssoId(ue.Supi-numPolId) as key
+       GroupIds                  []string
+       PolAssociationIDGenerator uint32
 }

 type UeAMPolicyData struct {
```

Reply:
done